mmseg/apis/ote/sample/sample.py

```diff
@@ -124,34 +124,41 @@ def main(args):
         openvino_task = openvino_task_cls(environment)
 
         logger.info('Get predictions on the validation set')
-        predicted_validation_dataset = openvino_task.infer(validation_dataset.with_empty_annotations(),
-                                                           InferenceParameters(is_evaluation=True))
-        resultset = ResultSetEntity(model=output_model,
-                                    ground_truth_dataset=validation_dataset,
-                                    prediction_dataset=predicted_validation_dataset)
+        predicted_validation_dataset = openvino_task.infer(
+            validation_dataset.with_empty_annotations(),
+            InferenceParameters(is_evaluation=True))
+        resultset = ResultSetEntity(
+            model=output_model,
+            ground_truth_dataset=validation_dataset,
+            prediction_dataset=predicted_validation_dataset,
+        )
         logger.info('Estimate quality on validation set')
         openvino_task.evaluate(resultset)
         logger.info(str(resultset.performance))
 
         logger.info('Run POT optimization')
-        optimized_model = ModelEntity(dataset,
-                                      environment.get_model_configuration(),
-                                      model_status=ModelStatus.NOT_READY)
-        openvino_task.optimize(OptimizationType.POT,
-                               dataset.get_subset(Subset.TRAINING),
-                               optimized_model,
-                               OptimizationParameters())
+        optimized_model = ModelEntity(
+            dataset,
+            environment.get_model_configuration(),
+            model_status=ModelStatus.NOT_READY)
+        openvino_task.optimize(
+            OptimizationType.POT,
+            dataset.get_subset(Subset.TRAINING),
+            optimized_model,
+            OptimizationParameters())
 
         logger.info('Get predictions on the validation set')
-        predicted_validation_dataset = openvino_task.infer(validation_dataset.with_empty_annotations(),
-                                                           InferenceParameters(is_evaluation=True))
-        resultset = ResultSetEntity(model=optimized_model,
-                                    ground_truth_dataset=validation_dataset,
-                                    prediction_dataset=predicted_validation_dataset)
+        predicted_validation_dataset = openvino_task.infer(
+            validation_dataset.with_empty_annotations(),
+            InferenceParameters(is_evaluation=True))
+        resultset = ResultSetEntity(
+            model=optimized_model,
+            ground_truth_dataset=validation_dataset,
+            prediction_dataset=predicted_validation_dataset,
+        )
         logger.info('Performance of optimized model:')
         openvino_task.evaluate(resultset)
         logger.info(str(resultset.performance))
 
-
```

Comment:
Why do you not follow PEP8 (two lines between functions)?

Reply:
I know that changing indent and code formatting is not really good idea, but at the moment I need compare 3 samples from 3 repositories, so (I'm really sorry) I just removed your changes in code formatting and made the formatting the same as in mmdetection sample to simplify my comparison  
(I use mmdetection sample as the reference, since it is the cutting edge, since Pavel checks OTE SDK changes on it).

Reply:
Ok, if it's so important.